lib/connections/service.go

```diff
@@ -441,6 +441,8 @@ func (s *service) handleHellos(ctx context.Context) error {
 		// connections are limited.
 		rd, wr := s.limiter.getLimiters(remoteID, c, c.IsLocal())
 
+		registerDeviceMetrics(remoteID.String())
```

Comment:
How about doing this in `func (*service) CommitConfiguration()`, which is called with the config (incl. list of remote devices) at startup and when the config changes?

Reply:
Not 100% sure if I put it to the correct place :sweat_smile: 

I've additionally added a call to remove the metric if the device got deleted from the config. But I just noticed that this probably won't work because the connections are handled at a later point... Where would I best put the metric deletion part?

Reply:
@calmh have you missed my last comment? The metric deletion probably isn't critical but still worth looking into I think.